core/azure-core/src/samples/java/com/azure/core/implementation/service/ServiceUtilJavaDocCodeSnippets.java

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.implementation.service;
+
+import com.azure.core.util.Context;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+/**
+ * Code snippets for {@link ServiceUtil}
+ */
+public class ServiceUtilJavaDocCodeSnippets {
+
+    /**
+     * Method to show how users can pass in context using {@link reactor.util.context.Context Reactor Context}
+     * for api calls that return single entity
+     */
+    public void serviceWithSingleResponse() {
+        ServiceUtilJavaDocCodeSnippets azureClient = this;
+        // BEGIN: com.azure.core.implementation.service.serviceutil.usersamplesingle
+        Mono<String> response = azureClient
+            .clientLibraryApiReturnsSingleItem("Hello, ")
+            .subscriberContext(
+                reactor.util.context.Context.of("FirstName", "Foo", "LastName", "Bar"));
```

Comment:
I would like to see the code samples not be fully-qualified like this. You can possibly invert it so that the Azure Context is fully-qualified (I don't think it is in a code sample).